Sources/Pageboy/PageboyViewController.swift

```diff
@@ -42,6 +42,30 @@ open class PageboyViewController: UIViewController {
         }
     }
     
+    public var minimumNumberOfTouches: Int? {
+        get {
```

Comment:
To maybe make this more simple (so that setting to `nil` doesn't leave a stale value) - this might be better to be an `Int` rather than `Int?` and just returning `1` (the system default) as a hardcoded default value if the `pageViewController` is `nil`.